oeps/oep-0032-arch-unique-identifier-for-users.rst

```diff
@@ -98,6 +98,8 @@ A consequence of the decision to use the *LMS user_id* as the unique user id is
 
 * The `LMS OAuth Dispatch App`_ will now include the *LMS user_id* in the JWTs (JSON Web Tokens) used as part of the `Open edX Authentication`_ upon request.
 
+* When exposing user information to other users, like in the case of a public Profile page, the associated *LMS user_id* should not be shared with other users.
```

Comment:
@nasthagiri: FYI: I added this warning text.  Not sure if it will do anything, but maybe it will help someone think about this somewhere it is needed.

Reply:
Could you elaborate on the rationale for adding this warning text? It seems like one's *LMS user_id* is equally as sensitive their username, and, as such, should be considered essentially public.

Reply:
@kdmccormick: The `LMS user_id` is not considered PII, so it is not equally sensitive.  Given that, this note was updated and you can let me know if it is more clear.  Thanks.

Reply:
Ah, I see. It's the _association_ that we don't want to make public. Makes sense.